### DISCUSSION TITLE
Airflow RPC Server

### DISCUSSION THREAD

**Discussion by hadarsharon** [apache/airflow#41936] [Q&A] [answered]
Hi,

I've noticed that the recent version of the Airflow Helm Chart allows for one to enable a component called RPC Server - in the Helm chart it appears with a leading underscore, which tells me it's something internal and/or private. 

Moreover, it seems like it calls `airflow internal-api` as its entrypoint, which is not yet supported and raises an error:

> airflow command error: argument GROUP_OR_COMMAND: invalid choice: 'internal-api' (choose from 'celery', 'cheat-sheet', 'config', 'connections', 'dag-processor', 'dags', 'db', 'info', 'jobs', 'kerberos', 'kubernetes', 'plugins', 'pools', 'providers', 'roles', 'rotate-fernet-key', 'scheduler', 'standalone', 'sync-perm', 'tasks', 'triggerer', 'users', 'variables', 'version', 'webserver'), see help above.

I'm guessing this isn't ready for use yet? Can someone shed a light on what is the purpose of this component and when/if it should be used?

Cheers

**Comment by potiuk** [comment]
Your guess is right,

**Reply by potiuk** [reply]
Not ready and not going to be enabled in the future. You can read more in the devlist discussion, searching for AIP-44

**Reply by potiuk** [reply]
Searchable archive of the devlist is here https://lists.apache.org/list.html?dev@airflow.apache.org

**Reply by potiuk** [reply, marked as answer]
Main discussion here: https://lists.apache.org/thread/b5ycrsmjw4b3tn2jpc368ck2l44vm53p

**Reply by hadarsharon** [reply]
Cheers, @potiuk 🥇
